examples/api/custom_projection_example.py

```diff
@@ -291,12 +291,12 @@ class DegreeFormatter(Formatter):
         """
 
         def __init__(self, round_to=1.0):
+            super(HammerAxes.DegreeFormatter, self).__init__()
             self._round_to = round_to
 
-        def __call__(self, x, pos=None):
+        def format_for_tick(self, x, pos=None):
```

Comment:
To keep Ticker etc a callable object, either format_for_tick() method could be invoked in the __call__(). 

The other option is to document carefully what the __call__ method does. The only downside is that this documentation is not directly obvious for a derived class if that class is not documented properly.